languages/emacs-lisp/reference/exercise-concepts/rna-transcription.md

```diff
@@ -0,0 +1,44 @@
+# Required
+
+- require: in the template
+- provide: in the template
+- defun: represents the code
+- returning values from functions
+- passing parameters to defuns
+- string: represents input
+
+# Common
+
+- lists: a string is a list
+- calling other functions: usually one defines new functions
+- mapconcat/mapcar/concat/dolist/: to process letters in a string
+- lambdas: people usually do the transcoding in a lamba on a seq-map
+
+# Style
+
+- cl-labels: defining functions inline
+- namespaces: or the lack of namespaces... how to name your functions properly
```

Comment:
Is this a Concept, or does this not exist?

Reply:
Maybe it should be `naming conventions`

Reply:
I changed the text